test/e2e/config/fixture.yaml

```diff
@@ -3,7 +3,7 @@ images:
 - name: quay.io/metal3-io/baremetal-operator:e2e
   loadBehavior: tryLoad
 # Save some time and network by using cached images if available
-- name: quay.io/metal3-io/baremetal-operator:release-0.4
+- name: quay.io/metal3-io/baremetal-operator:release-0.8
```

Comment:
from release-0.4 to release-0.8 ? Why the jump? Were we doing things wrongly before?

Reply:
It is preloading images. We don't use 0.4 anymore, and I guess forgot to bump it. I figured 0.8 is most used, so preloading it helps the most?